svc/WebSocketService.ts

```diff
@@ -361,6 +400,8 @@ export interface WebSocketTelemetry {
         connError?: WebSocketEventTelemetry;
         msgReceived?: WebSocketEventTelemetry;
         msgSent?: WebSocketEventTelemetry;
+        heartbeatReceived?: WebSocketEventTelemetry;
+        heartbeatSent?: WebSocketEventTelemetry;
```

Comment:
As per commit msg comment - seemed useful and cheap to give these dedicated counters.  Heartbeats are included in the general msgSent/Received counters, but so are app-level messages of an unpredictable volume.  I want to see how big the diff between these two numbers is in practice for clients - they should be equal or (very briefly!) within 1 of each other.